components/admin_real/desktop/ReservationTable.jsx

```diff
@@ -62,7 +36,7 @@ export default function ReservationTable() {
           fetchReservations({
             rangeInitial: initial,
             rangeEnd: end,
-            diveShopId: diveShop.id,
+            diveShopId: context.diveShop.id,
           }).then(({ data, error, count }) => {
             setData(data);
```

Comment:
we have a problem, you are using context but then you are not using it here. This is going to create a mismatch. If we move to context, we should have the fetch function in the context, and utilize it here.

This has a good example https://betterprogramming.pub/build-a-redux-like-store-with-react-context-hooks-234e3774495f